plugins/browser-plugin-snowplow-ecommerce/test/events.test.ts

```diff
@@ -354,4 +357,36 @@ describe('SnowplowEcommercePlugin events', () => {
       data: { type: 'refund' },
     });
   });
+
+  it('trackTransactionError adds the expected "transaction_error" event to the queue', () => {
+    const transactionError: Omit<TransactionError, 'transaction'> = {
+      resolution: 'rejection',
+      error_code: 'E123',
+      error_shortcode: 'CARD_DECLINE',
+      error_type: 'hard',
+    };
+    const transaction: Transaction = { revenue: 45, currency: 'EUR', transaction_id: '12345', payment_method: 'card' };
```

Comment:
Since we're importing this type now should we update the test for `trackTransaction` to use it too?